### PR TITLE
Add a robots.txt file

### PR DESCRIPTION
**Pull request by tomodwyer**
To stop crawlers from trying (and failing) to access certain areas of the site.

### CODE REVIEW

**Comment by ghickman** [comment]
Fixes #1036

**Comment by madwort** [comment]
@ghickman did you mean #1035 ??

File: static/robots.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+User-agent: *
+Disallow: /api/
+Disallow: /applications/
+Disallow: /apply/
```
Comment (sebbacon) [inline static/robots.txt:4]: Is this definitely right? Might be nice for people to be able to find us via IG-related keywords?

**Comment by ghickman** [comment]
> @ghickman did you mean #1035 ??

Yes 🤦‍♂️🤦‍♂️🤦‍♂️

File: static/robots.txt (hunk 1 above)
Comment (tomodwyer) [reply static/robots.txt:4]: I am making a ticket to remove these once we have completed our voyage. I just wanted to block them from crawlers before they are "ready".

**Pull request merged**